### PR TITLE
Remove create transaction in worker pool

### PR DESCRIPTION
**Pull request by ygao76**
## Summary
- Replace fixture create transaction with raw transaction and post()
- Remove createTransaction
- Use size constants for hardcoded values
## Testing Plan
unit test
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-rust-nodejs/index.d.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@ export interface NativeSpendDescription {
   nullifier: Buffer
 }
 export const PROOF_LENGTH: number
+export const TRANSACTION_SIGNATURE_LENGTH: number
```
Comment (NullSoldier) [inline ironfish-rust-nodejs/index.d.ts:36]: Can you split all of these changes out into another PR? They are not related to removing `workerPool.createTransaction()` right and they are fairly opinionated so mat or rohan should review them separately.

Comment (NullSoldier) [reply ironfish-rust-nodejs/index.d.ts:36]: That sounds good, but it's still unrelated to this PR. It's not needed to accomplish the PR task. It should be pretty easy to split this out into a different PR.

Comment (rohanjadvani) [reply ironfish-rust-nodejs/index.d.ts:36]: Yeah @ygao76 I think this is a sensible change; I'm all for removing magic numbers. But as @NullSoldier mentioned, we want to keep the code changes in this PR focused on just the worker pool. I'd recommend keeping this refactoring but just cherry picking those changes out in a new PR.

**Comment by ygao76** [comment]
PTAL @NullSoldier

**Review by NullSoldier — APPROVED** [review]

**Comment by NullSoldier** [comment]
Thanks, looks good!

**Pull request merged**